### ISSUE TITLE
Add term bartholin cyst

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10330]
**Preferred term label:**
Bartholin cyst

**Synonyms**
Vulvar cyst, Bartholin's duct cyst

**Definition (free text, please give PubMed ID)**
PMID: 30335304: usually a unilateral, asymptomatic blockage of the Bartholin gland, which could lead to a painless swelling.

**Parent term (use hpo.jax.org/app)**
Abnormal labia majora morphology HP:0012881

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
PMID: 36760371
ICD10: N75
https://www.icd10data.com/ICD10CM/Codes/N00-N99/N70-N77/N75-/N75.0#:~:text=Cyst%20of%20Bartholin's%20gland,-2016%202017%202018&text=Code%20Female%20Dx-,N75.,ICD%2D10%2DCM%20N75.

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
added as HP:0430134
